source/welcome/about-teams.rst

```diff
@@ -0,0 +1,156 @@
+
+1. Select the Team menu at the top of the Navigation Panel.
+2. Select **Invite People**.
+3. If there's an option to invite members and guests, choose **Invite Members**.
+4. Select the **Copy Link** button to save the URL to your clipboard and share it with those you want to invite to the team.
+
+Adding someone to your team
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If someone you want to add to your team already has an account on the server, you can add them to your team without sending an invitation.
+
+**Add someone to your team**
+
+1. Select the Team menu at the top of the Navigation Panel.
+2. Select **Invite People**.
+3. Choose **Invite Members** or **Invite Guests**.
+4. Search for the person you want to add. If the people you want to add are not on the list, then they do not have an account on your server.
+5. When adding a guest, you must specify the channels the guest can access.
+6. Select **Invite Members** or **Invite Guests**.
+
+.. note:: 
+   
+   - If you aren't able to see the **Invite Guests** option, confirm with your System Admin that you have the correct permissions.
+   - System Admins can view and add team membership to individual users on the user profile page (via **System Console > Users > User Profile**) by selecting **Add Team**.
+
+
+Remove people from teams
+------------------------
+
+A Team Admin can remove a user from a team via **Team menu > Manage Members > Remove From Team** in the dropdown menu beside a user entry.
+
+When a user is removed from a team, the team will no longer show up in their team sidebar. If they currently have the team open, they are redirected to the first team that appears on their team sidebar. If they didn't belong to any other teams, the user is sent to the team selection page.
+
+Removing a user from the team does not deactivate the account. The user will still be able to sign in to the site, and join other teams. They will also be able to rejoin the team they were removed from if they receive another invite, or if the team is set to `"Allow any user with an account on this server to join this team" <https://docs.mattermost.com/welcome/team-settings.html#allow-anyone-to-join-this-team>`__. If the user does rejoin the team, they will no longer belong to the channels they were previously a part of, and they will lose all Admin privileges if they had them previously.
+
+A System Admin can also remove users from teams via **System Console > Users**, and selecting the dropdown beside a user entry and selecting **Manage Teams**. The list of teams an individual user belongs to can be viewed on the user's profile page via **System Console > Users** and selecting the member's name from the list provided in the **User Configuration** screen.
+
+Leave a team
+------------
+
+Users can also choose to remove themselves from a team, from **Team menu > Leave Team**. This will remove the user from the team, and from all Public channels and Private channels on the team.
```

Comment:
```suggestion
Users can also choose to remove themselves from a team, from **Team menu > Leave Team**. This will remove the user from the team, and from all public channels and private channels on the team.
```